src/scenic/simulators/carla/model.scenic

```diff
@@ -70,7 +70,8 @@ except ModuleNotFoundError:
     class _CarlaVehicle: pass
     class _CarlaPedestrian: pass
 
-param carla_map = None
+map_town = ((str(globalParameters.map).split('/'))[-1]).split('.')[0]
+param carla_map = map_town if globalParameters.map else None
```

Comment:
What's the reason for the check here? If `map` is the empty string, an error will have already been raised when we tried to open the map file.

Reply:
Ack! I remove the redundant checks for setting `map_town` here!